### PR TITLE
Add `api_kwargs` to `log_out`

### PR DESCRIPTION
**Pull request by Bibo-Joshi**
### Checklist for PRs

- [x] Added `.. versionadded:: version`, `.. versionchanged:: version` or `.. deprecated:: version` to the docstrings for user facing changes (for methods/class descriptions, arguments and attributes)
- [x] Created new or adapted existing unit tests
- [ ] Documented code changes according to the [CSI standard](https://standards.mousepawmedia.com/en/stable/csi.html)
- [ ] Added myself alphabetically to `AUTHORS.rst` (optional)
- [ ] Added new classes & modules to the docs and all suitable `__all__` s

### CODE REVIEW

File: tests/test_bot.py [hunk 1]
```diff
@@ -2964,3 +2965,18 @@ def test_coroutine_functions(self):
             "The following methods should be defined as coroutine functions: "
             f"{','.join(non_coroutine_functions)} "
         )
+
+    def test_api_kwargs_present(self):
```
Comment (harshil21) [inline tests/test_bot.py:2969]: why not additionally check for the timeout kwargs? Or are we already testing that somewhere

**Review by harshil21 — COMMENTED** [review]
didn't I point this out in some PR? Maybe the change was never implemented or was lost in a rebase

File: tests/test_bot.py (hunk 1 above)
Comment (Bibo-Joshi) [reply tests/test_bot.py:2969]: Don't recall that :D but I'll extend the tests 👍

File: tests/test_bot.py [hunk 2]
```diff
@@ -2945,22 +2956,35 @@ def test_camel_case_bot(self):
                 not_available_camelcase_functions.append(function_name)
```
Comment (harshil21) [inline tests/test_bot.py:2956]: can `test_camel_case_bot` also make use of the decorator

**Review by harshil21 — APPROVED** [review]

**Pull request merged**